source/configuration_manual/index.rst

```diff
@@ -45,7 +45,6 @@ Configuration Manual
    quota_clone_plugin
    quota/index
    quota_plugin
-   replication
```

Comment:
This needs to be in the previous commit, otherwise that would yield errors and not produce any viable output. (Same as the other comment.)